tests/Javascript/Components/Form/MultiSelect.spec.js

```diff
@@ -0,0 +1,83 @@
+import {beforeEach, afterEach, expect, jest, test} from "@jest/globals";
+
+jest.mock('laravel-jetstream')
+
+import {createLocalVue, mount, shallowMount} from '@vue/test-utils'
+import {InertiaApp} from '@inertiajs/inertia-vue'
+import {InertiaForm} from 'laravel-jetstream'
+import Component from '@src/Components/Form/MultiSelect'
+
+let localVue, wrapper;
+
+beforeEach(() => {
+    localVue = createLocalVue()
+    localVue.use(InertiaApp)
+    localVue.use(InertiaForm)
+
+  wrapper = shallowMount(Component, {localVue, propsData: {
+    options: {
+      1: 'a',
+      9: 'b',
+      6: 'c',
+    },
+    selectedOnInit: {
+      1: 'a',
+    }
+  }})
+
+});
+
+afterEach(() => {
+  wrapper = null;
+})
+
+test('should mount without crashing', () => {
+  expect(wrapper.text()).toBeDefined();
```

Comment:
Should you be mounting or shallow mounting the component here?

Reply:
I figured its best to mount it in the beforeEach section, since we reuse the same props and data for most of the tests in this file

Reply:
Yes makes sense, thanks. Probably a cleaner way of doing it as well.

Reply:
I think so too.